Apps/Encounter/src/Models/HospitalVisitModel.cs

```diff
@@ -27,7 +28,7 @@ public class HospitalVisitModel
         /// Gets or sets the encounter id.
         /// </summary>
         [JsonPropertyName("encounterId")]
-        public string? EncounterId { get; set; }
+        public string EncounterId { get; set; } = string.Empty;
```

Comment:
Why was this (C# and Typescript) changed from nullable to default empty string but other attributes were kept nullable?